### PR TITLE
Laboratory work #4, Arina Savina - 22FPL1

### PR DESCRIPTION
**Pull request by a00rina**

### CODE REVIEW

File: lab_4_fill_words_by_ngrams/main.py [hunk 1]
```diff
@@ -28,6 +30,17 @@ def _tokenize(self, text: str) -> tuple[str, ...]:  # type: ignore
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not isinstance(text, str) or not text:
+            raise ValueError
```
Comment (artyomtugaryov) [inline lab_4_fill_words_by_ngrams/main.py:34]: add message

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_4_fill_words_by_ngrams/main.py [hunk 2]
```diff
@@ -39,6 +52,10 @@ def _put(self, element: str) -> None:
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not isinstance(element, str) or not element:
+            raise ValueError
```
Comment (artyomtugaryov) [inline lab_4_fill_words_by_ngrams/main.py:56]: message

**Review by artyomtugaryov — COMMENTED** [review]
add messages to errors

**Comment by vitazaytseva** [comment]
Well done! Good job!

**Pull request closed without merging**